### DISCUSSION TITLE
Shortcut for toggling propagate track labels

### DISCUSSION THREAD

**Discussion by Marielamend** [talmolab/sleap#1638] [Ideas]
Enhancement: Adding a shortcut for toggling propagate track labels, the idea came up when discussing bug #1637!

**Comment by gitttt-1234** [comment]
Closing this discussion, added in https://github.com/talmolab/sleap/pull/2714

**Comment by talmo** [comment]
Implemented in #2714 (merged to `develop`). The **Tracks > Propagate Track Labels** toggle now has a keyboard shortcut — default key `P` — and is rebindable via the Keyboard Shortcuts dialog. Thanks for the suggestion, @Marielamend!
